templates/docs/examples/index.html

```diff
@@ -7,15 +7,15 @@
 {% block title %}Component examples{% endblock %}
 
 {% block body %}
-<div class="p-strip is-bordered">
+<main id="main-content" class="p-strip is-bordered">
   <div class="u-fixed-width">
-    <h2>Component examples</h2>
+    <h1 class="p-heading--2">Component examples</h1>
```

Comment:
I know that changing the look of the headings is not meant to happen in this PR, but I wonder that while we update headings to their proper levels, maybe their default sizes will be fine? So we could avoid adding heading class names to change heading sizes?

I would suggest stripping these headings of `p-heading-` class names and getting design review if we are OK with how updated sizes look.

This refers to all pages updated in this PR.

Reply:
That's a fair point - and would making adjusting the heading levels on individual examples easier too. They're all in markdown, and they all assume there's a h1 somewhere else, so all headings are one level too low.

Reply:
@lyubomir-popov could you take a look at this, with respect to the above comments?

Reply:
The heading hierarchy is currently very inconsistent, ans I see no point in forcing the use of modifier classes until we standardize it. I'm ok with dropping those and usng whatever the actual hierarchy requires. We'll endeavor to address this in the design system redesign.

Reply:
Here's an example of inconsistency based on heading length:
![image](https://user-images.githubusercontent.com/2741678/123671234-f5954980-d835-11eb-9155-d73886d73ab0.png)